### PR TITLE
/search (refactored), /logs, /xkcd, /course, and /markdown (refactored)

### LINKED ISSUE #69: Wikipedia extract often irrelevant
When searching something which is not a broad topic, there tends to not be a wikipedia article about it. However the bot searches wikipedia too broadly, making the message less helpful and half off topic. e.g.
image
The wiki extract is completely off topic and too broad to be useful. If we can't come up with a way to accurately determine the usefulness of the wiki extract on a case by case basis (one possible suggestion might be to check if the wiki link is within the first k google search results) then the wiki extract should be removed entirely.

### PR DESCRIPTION
**Pull request by abrahammurciano**
- removed the ":continue" special search, if they want to read more they can click the link, also I don't think anyone actually used it.
- resolves #69: only shows wiki extract when relevant (if a Wikipedia link is in one of the first 10 google results)

### CODE REVIEW

**Comment by abrahammurciano** [comment]
Changed ++logs to /logs

**Comment by abrahammurciano** [comment]
Changed ++xkcd to /xkcd get (this is for random and id), /xkcd search, and /xkcd latest

**Comment by abrahammurciano** [comment]
changed ++addcourse to /course add
changed ++deletecourse to /course delete

**Comment by abrahammurciano** [comment]
changed ++markdown to /markdown
uses unicode fonts to show preview as suggestions while typing the commands
(no longer needs aliases, since they are options to be selected)

File: modules/xkcd/cog.py [hunk 1]
```diff
@@ -1,58 +1,76 @@
...
+			create_option(
+				name="query",
+				description="Your search terms",
+				option_type=SlashCommandOptionType.STRING,
+				required=True,
+			)
+		],
+	)
+	async def xkcd_search(self, ctx: SlashContext, query: str):
+		comic = self.xkcd_fetcher.search_relevant(query)
```
Comment (DenverCoder1) [inline modules/xkcd/cog.py:72]:
```suggestion
		await ctx.defer()
		comic = self.xkcd_fetcher.search_relevant(query)
```

**Review by DenverCoder1 — APPROVED** [review]

**Pull request merged**